pyramid_hypernova/batch.py

```diff
@@ -58,11 +58,11 @@ def __init__(self, batch_url, plugin_controller, pyramid_request, max_batch_size
         self.json_encoder = json_encoder
         self.pyramid_request = pyramid_request
 
-    def render(self, name, data):
+    def render(self, name, data, context={}):
```

Comment:
we don't want to use mutable default kwargs in python, because the same value is used every time the function is called (i.e. if anything mutates that context dict, it will still be mutated in the next call because it's the same object)

instead usually we write:

```python
def render(self, name, data, context=None):
    if context is None:  # pragma: no cover
        context = {}
```

Reply:
another possibility might be to just have it default to None? since we never access it (only plugins or whatever access it), maybe we should just let the user choose the type?

Reply:
Thanks!

I think defaulting to {} is probably a good  thing, to make the shape of the object we send to hypernova consistent.